src/config/passport.js

```diff
@@ -0,0 +1,35 @@
+const passport = require("passport");
+const LocalStrategy = require("passport-local").Strategy;
+const sceServiceAPI = require("../services/at-sce-api.service");
+ 
+passport.use(
+    new LocalStrategy({
+            usernameField: "email",
+            passwordField: "password"
+        },
+        async(email, password, done) => {
+            // Match Email's User
+            const request = {
+                email : email,
```

Comment:
if they're named the same there's no need to do email: email with object = {email, password} should be enough